From ab1887e9983b1586f7353309ad8598a5861366c8 Mon Sep 17 00:00:00 2001
From: John Jacques <john.jacques@intel.com>
Date: Wed, 16 Nov 2016 18:56:59 +0200
Subject: [PATCH 178/278] axxia: Update the Axxia Device Trees for 5600 and
 6700

Add a mac-address entry to the nemac.
The mac-address needs to be set before kexec.

Signed-off-by: John Jacques <john.jacques@intel.com>
Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
---
 arch/arm64/boot/dts/intel/axc67xx.dtsi | 1 +
 arch/arm64/boot/dts/intel/axm56xx.dtsi | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/intel/axc67xx.dtsi b/arch/arm64/boot/dts/intel/axc67xx.dtsi
index 5a65714..0e703a3 100644
--- a/arch/arm64/boot/dts/intel/axc67xx.dtsi
+++ b/arch/arm64/boot/dts/intel/axc67xx.dtsi
@@ -110,6 +110,7 @@
 			interrupts = <GIC_SPI 0 IRQ_TYPE_LEVEL_HIGH>,
 				     <GIC_SPI 1 IRQ_TYPE_LEVEL_HIGH>;
 			dma-coherent;
+			mac-address = [00 00 00 00 00 00];
 			status = "disabled";
 		};
 
diff --git a/arch/arm64/boot/dts/intel/axm56xx.dtsi b/arch/arm64/boot/dts/intel/axm56xx.dtsi
index 748f19e..54e2e04 100644
--- a/arch/arm64/boot/dts/intel/axm56xx.dtsi
+++ b/arch/arm64/boot/dts/intel/axm56xx.dtsi
@@ -100,6 +100,7 @@
 			interrupts = <GIC_SPI 0 IRQ_TYPE_LEVEL_HIGH>,
 				     <GIC_SPI 1 IRQ_TYPE_LEVEL_HIGH>;
 			dma-coherent;
+			mac-address = [00 00 00 00 00 00];
 			status = "disabled";
 		};
 
-- 
1.8.5.2.233.g932f7e4

